src/python/pants/engine/internals/engine_test.py

```diff
@@ -198,20 +199,17 @@ def test_include_trace_error_raises_error_with_trace(self):
             dedent(
                 f"""
                 1 Exception encountered:
-                Computing Select(<{__name__}.B object at 0xEEEEEEEEE>, A)
-                  Computing Task({fmt_rust_function(nested_raise)}(), <{__name__}.B object at 0xEEEEEEEEE>, A, true)
-                    Throw(An exception for B)
-                      Traceback (most recent call last):
-                        File LOCATION-INFO, in call
-                          val = func(*args)
-                        File LOCATION-INFO, in nested_raise
-                          fn_raises(x)
-                        File LOCATION-INFO, in fn_raises
-                          raise Exception(f"An exception for {{type(x).__name__}}")
-                      Exception: An exception for B
+
+                Traceback (most recent call last):
+                  File LOCATION-INFO, in call
+                    val = func(*args)
+                  File LOCATION-INFO, in nested_raise
+                    fn_raises(x)
+                  File LOCATION-INFO, in fn_raises
+                    raise Exception(f"An exception for {{type(x).__name__}}")
+                Exception: An exception for B
```

Comment:
This is much nicer. It also means that `pants.toml` contents won't render in the stacktrace anymore!